serialize.py

```diff
@@ -12,26 +11,25 @@
 
 @hydra.main(config_path="configs", config_name="serialize")
 def main(cfg: DictConfig) -> None:
-    # absolute path
     cfg.data.tokenizer_dir = to_absolute_path(cfg.data.tokenizer_dir)
     cfg.data.corpus_dir = to_absolute_path(cfg.data.corpus_dir)
 
     tokenizer = instantiate(cfg.model.tokenizer)
     dataset_builder = instantiate(cfg.model.dataset_builder, tokenizer=tokenizer)
     dataset = instantiate(cfg.data.load_dataset)
-    print(dataset[:2])
+
     dataset = dataset.map(
         **cfg.data.map,
         function=dataset_builder.make_examples,
         remove_columns=dataset.column_names,
         features=dataset_builder.features,
     )
-    print(dataset[:2])
+
     dst_name = tokenizer.__class__.__name__
     dst_name = dst_name[: dst_name.find("Tokenizer")].lower()
 
-    dataset_path = os.path.join(cfg.data.dataset_dir, dst_name)
-    dataset.save_to_disk(str(dataset_path))
+    dataset_path = to_absolute_path(os.path.join(cfg.data.dataset_dir, dst_name))
```

Comment:
```suggestion
    dataset_path = os.path.join(to_absolute_path(cfg.data.dataset_dir), dst_name)
```
같은 거지만 hydra config를 먼저 absolute_path로 변환한 뒤 join

Reply:
@bzantium 더 좋은 코드네요!